pkg/symbolizer/symbolizer.go

```diff
@@ -89,8 +114,10 @@ func (s *Symbolizer) runSymbolizationCycle(ctx context.Context) {
 			Limit:  s.batchSize,
 			MinKey: prevMaxKey,
 		})
+		s.attempts.Inc()
```

Comment:
So the counters actually count per batch. This is worth mentioning in the description.

Reply:
We should also consider this when we add the histogram for the duration.